appletun/__main__.py

```diff
@@ -0,0 +1,242 @@
+import plistlib
+import re
+import secrets
+import string
+from os import path
+from sys import platform
+from typing import IO, Optional
+
+import click
+import psutil
+from plumbum import local
+from pymobiledevice3.cli.cli_common import Command
+from pymobiledevice3.lockdown import LockdownServiceProvider
+from pymobiledevice3.services.mobile_config import MobileConfigService
+
+BASE_VPN_CONFIG_FORMAT = """
+conn {vpn_name}
+  left=0.0.0.0                  # Server listens on any IPv4 address
+  leftid={vpn_name}             # Identity presented by the server
+  leftsubnet=0.0.0.0/0,::/0     # Server-side networks (IPv4 and IPv6)
+  leftauth=psk                  # Server uses a pre-shared key
+
+  right=%any                    # Accept any client IP
+  rightid=%any                  # Accept any client identity
+  rightdns=%config4,%config6    # Push DNS settings (IPv4, IPv6)
+  rightsubnet=%dynamic          # Dynamically assign subnets to clients
+  rightsourceip=133.33.37.0/24,fec3:1337::0/112   # IPv4/IPv6 pool for clients
+  rightauth=psk                 # Client also uses a pre-shared key
+
+  ike=aes256-sha256-ecp256,aes256-sha256-modp2048!     # Phase 1 proposals
+  esp=aes256-sha256-ecp256,aes256-sha256-modp2048!     # Phase 2 proposals
+  keyexchange=ikev2             # Use IKEv2 protocol
+  auto=add                      # Load connection at startup
+"""
+
+PROFILE_UUID = "B6CFC043-A872-4EE1-BADF-53471B910BA7"
+
+
+def get_machine_ips(ipv4: bool = True,
+                    ipv6: bool = True):
+    """ Retrieve all ip addresses belonging to the current machine (excluding loopback) """
+    ifaces = psutil.net_if_stats()
+    addresses = []
+    for iface, addrs in psutil.net_if_addrs().items():
+        if 'loopback' in ifaces[iface].flags:
+            continue
+        for addr in addrs:
+            if ipv4 and addr.family.name == 'AF_INET':  # IPv4
+                addresses.append(addr.address)
+            elif ipv6 and addr.family.name == 'AF_INET6':  # IPv6
+                addresses.append(addr.address)
+
+    return addresses
+
+def get_vpn_config_dir():
+    """ Locate VPN config dir """
+    if "darwin" == platform:
+        return path.join(local['brew']('--prefix').strip(), 'etc')
```

Comment:
Use `pathlib` instead.
```suggestion
        return Path(local['brew']('--prefix').strip()) / 'etc'
```